lib/sen_tweet_web/templates/page/stream-admin.html.eex

```diff
@@ -0,0 +1,16 @@
+<nav class="level">
```

Comment:
The [Bulma](https://bulma.io/) framework is already added to this project so you can use its css classes to build whatever UI you like.